docs/acp/user/intro-to-acp/developing-an-app.md

```diff
@@ -0,0 +1,142 @@
+---
+title: Developing an app
+Descriptions: Learn how to develop an app
+template: howto-guide-template
+---
+
+To develop an app, follow the instructions in this document.
+
+## Prerequisites
+
+- You have completed the [thought process](#thought-process) for your app.
+- You have [installed the Spryker SDK](https://docs.spryker.com/docs/sdk/dev/spryker-sdk.html#installation).
+- You have an empty GitHub repository.
+- You have a local project directory where you want to work, for example, `/www/my-app`.
+ 
+## Thought process
+First, think about what your app should be capable of: what features it will bring, and what data will be exchanged, not only to you but also to those interested in your app functionality. For example, what messages could be of interest to others, and what API endpoints you should provide. 
+
+### API-first
+It's strongly recommended that apps follow the API-first approach. 
+API-first means that your app is centered on the API. It should be possible to perform every action via the scripting language, and every piece of functionality should be available for other systems to leverage. For more information on the API-first approach, see [this blog post](https://www.algolia.com/blog/product/the-5-principles-of-api-first-development-and-what-does-api-first-even-mean/).
```

Comment:
did not get what it means "possible to perform every action via the scripting language"...
maybe via API endpoint ? 

@stereomon @lenadoc

Reply:
That is a text provided by Algolia.